src/Sylius/Bundle/CoreBundle/spec/Sitemap/Renderer/TwigAdapterSpec.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace spec\Sylius\Bundle\CoreBundle\Sitemap\Renderer;
+ 
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapInterface;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapUrlInterface;
+use Symfony\Bundle\TwigBundle\TwigEngine;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class TwigAdapterSpec extends ObjectBehavior
+{
+    function let(TwigEngine $twig)
```

Comment:
And what about name? Can we came up with better name for ``EngineInterface`` than ``$twig``? ;)